src/enums/geo/epsg.ts

```diff
@@ -0,0 +1,5 @@
+export enum EPSG {
```

Comment:
should talk with @syncush about it

Reply:
Think this one should be proxied (as an example for type proxy principles)
`import { epsg } from 'EPSG_LIST_PKG';`

`export type Projections = epsg;`

`export const ExportProjections = {`
`  "EPSG:4326": epsg["EPSG:4326"],`
`  "EPSG:3857": epsg["EPSG:3857"],`
`  ...`
`};`

Reply:
merged your branch with the fix